<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poisson Matting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionVideo"/>
    <addaction name="actionPhoto"/>
    <addaction name="actionCamera"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlay"/>
  </widget>
  <widget class="QToolBar" name="playerToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVideo"/>
   <addaction name="actionPhoto"/>
   <addaction name="actionCamera"/>
   <addaction name="separator"/>
   <addaction name="actionMat"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
  </widget>
  <widget class="QToolBar" name="timelineToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/38-airplane.png</normaloff>:/icons/icons/38-airplane.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/48-fork-and-knife.png</normaloff>:/icons/icons/48-fork-and-knife.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/70-tv.png</normaloff>:/icons/icons/70-tv.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionPhoto">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/42-photos.png</normaloff>:/icons/icons/42-photos.png</iconset>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </action>
  <action name="actionMat">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/183-genie-lamp.png</normaloff>:/icons/icons/183-genie-lamp.png</iconset>
   </property>
   <property name="text">
    <string>Mat</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/86-camera.png</normaloff>:/icons/icons/86-camera.png</iconset>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
